frontend/packages/ux-editor/src/App.tsx

```diff
@@ -29,6 +32,7 @@ export function App() {
   const { isSuccess: isDatamodelFetched, isError: dataModelFetchedError } =
     useDatamodelMetadataQuery(org, app);
   const { isSuccess: areTextResourcesFetched } = useTextResourcesQuery(org, app);
+  const { data: appVersion, isSuccess: isAppVersionFetched } = useAppVersionQuery(org, app);
```

Comment:
Ref. prev comment. 😊 
```suggestion
  const { data: appVersion } = useAppVersionQuery(org, app);
```